projects/demo/src/modules/tables/table/examples/5/index.ts

```diff
@@ -0,0 +1,72 @@
+import {Component} from '@angular/core';
+import {changeDetection} from '@demo/emulate/change-detection';
+import {encapsulation} from '@demo/emulate/encapsulation';
+import {TuiComparator} from '@taiga-ui/addon-table';
+import {toInt, TuiDay} from '@taiga-ui/cdk';
+
+interface User {
+    readonly name: string;
+    readonly dob: TuiDay;
+}
+
+const TODAY = TuiDay.currentLocal();
+const FIRST = [
+    'John',
+    'Jane',
+    'Jack',
+    'Jill',
+    'James',
+    'Joan',
+    'Jim',
+    'Julia',
+    'Joe',
+    'Julia',
+];
+
+const LAST = [
+    'Smith',
+    'West',
+    'Brown',
+    'Jones',
+    'Davis',
+    'Miller',
+    'Johnson',
+    'Jackson',
+    'Williams',
+    'Wilson',
+];
+
+const DATA: readonly User[] = Array.from({length: 300}, () => ({
+    name: `${LAST[Math.floor(Math.random() * 10)]}, ${
+        FIRST[Math.floor(Math.random() * 10)]
```

Comment:
please add this example to cypress exclusions

https://github.com/Tinkoff/taiga-ui/blob/d1b22cccdf65b3644be670208719834b0fe53e2d/projects/demo-integrations/cypress/support/exclusions.ts#L5